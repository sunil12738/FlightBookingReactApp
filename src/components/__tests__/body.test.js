import {shallow} from 'enzyme';
import React from 'react';
import Body from '../body.jsx';

test('Link changes the class when hovered', () => {
	const component = shallow(<Body />);
	expect(component).toMatchSnapshot();
});