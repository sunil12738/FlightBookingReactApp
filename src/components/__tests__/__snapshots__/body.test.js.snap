// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className=""
>
    <div
        className="float-left"
    >
        <LeftPane />
    </div>
    <div
        className="float-right"
    >
        <RightPane />
    </div>
</div>,
  "nodes": Array [
    <div
      className=""
>
      <div
            className="float-left"
      >
            <LeftPane />
      </div>
      <div
            className="float-right"
      >
            <RightPane />
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Body />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance":  {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className=""
>
          <div
                    className="float-left"
          >
                    <LeftPane />
          </div>
          <div
                    className="float-right"
          >
                    <RightPane />
          </div>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className=""
>
          <div
                    className="float-left"
          >
                    <LeftPane />
          </div>
          <div
                    className="float-right"
          >
                    <RightPane />
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Body />,
}
`;
